crates/forge/tests/cli/cmd.rs

```diff
@@ -1532,34 +1530,127 @@ contract ContractThreeTest is DSTest {
         ..Default::default()
     });
 
-    let first_out = cmd
-        .forge_fuse()
+    cmd.forge_fuse().arg("test").arg("--gas-report").assert_success().stdout_eq(str![[r#"
+...
+| src/Contracts.sol:ContractOne contract |                 |       |        |       |         |
+|----------------------------------------|-----------------|-------|--------|-------|---------|
+| Deployment Cost                        | Deployment Size |       |        |       |         |
+| 103375                                 | 255             |       |        |       |         |
+| Function Name                          | min             | avg   | median | max   | # calls |
+| foo                                    | 45387           | 45387 | 45387  | 45387 | 1       |
+
+
+| src/Contracts.sol:ContractThree contract |                 |        |        |        |         |
+|------------------------------------------|-----------------|--------|--------|--------|---------|
+| Deployment Cost                          | Deployment Size |        |        |        |         |
+| 103591                                   | 256             |        |        |        |         |
+| Function Name                            | min             | avg    | median | max    | # calls |
+| baz                                      | 260712          | 260712 | 260712 | 260712 | 1       |
+
+
+| src/Contracts.sol:ContractTwo contract |                 |       |        |       |         |
+|----------------------------------------|-----------------|-------|--------|-------|---------|
+| Deployment Cost                        | Deployment Size |       |        |       |         |
+| 103375                                 | 255             |       |        |       |         |
+| Function Name                          | min             | avg   | median | max   | # calls |
+| bar                                    | 64984           | 64984 | 64984  | 64984 | 1       |
+...
+
+"#]]);
+    cmd.forge_fuse()
         .arg("test")
         .arg("--gas-report")
+        .arg("--json")
         .assert_success()
-        .get_output()
-        .stdout_lossy();
-    assert!(first_out.contains("foo") && first_out.contains("bar") && first_out.contains("baz"));
+        .stdout_eq(str![[r#"
+{"gas":103375,"size":255,"functions":{"foo":{"foo()":{"calls":1,"min":45387,"mean":45387,"median":45387,"max":45387}}}}
+{"gas":103591,"size":256,"functions":{"baz":{"baz()":{"calls":1,"min":260712,"mean":260712,"median":260712,"max":260712}}}}
+{"gas":103375,"size":255,"functions":{"bar":{"bar()":{"calls":1,"min":64984,"mean":64984,"median":64984,"max":64984}}}}
+
+
+"#]]);
```

Comment:
use .json_lines() on str

Reply:
Nice! Using `is_jsonlines()` (`.json_lines()` has been deprecated in favor of `is_jsonlines()`)